ni_measurementlink_service/session_management/_client.py

```diff
@@ -318,6 +318,57 @@ def unregister_multiplexer_sessions(
         )
         self._get_stub().UnregisterMultiplexerSessions(request)
 
+    def get_multiplexer_sessions(
```

Comment:
Please remind me why you are adding this API when `client.reserve_sessions()` + `reservation.multiplexer_session_info` does almost the same thing (but better, because the property returns the session objects and this method doesn't).

Is `get_multiplexer_sessions` an alternative to `reserve_sessions` for multiplexer-only usage?

I think the docstring should explain this so that users are less likely to use the wrong one.

Reply:
> Is get_multiplexer_sessions an alternative to reserve_sessions for multiplexer-only usage?

Yes, that's right. The plan in the TestStand fixture is to handle instrument and multiplexer session initialization separately. The `get_multiplexer_sessions` API will enable multiplexer registration as a standalone operation. Attaching the link to the pseudo-code presented in the HLD [here](https://dev.azure.com/ni/DevCentral/_git/ASW?path=/Specs/Test%20Automation%20Frameworks/F2430675_MuxInfoSupportForSessionManagerReserveFunction/F2430675_MuxInfoSupportForSessionManagerReserveFunction.md&version=GBmain&line=558&lineEnd=558&lineStartColumn=1&lineEndColumn=32&lineStyle=plain&_a=contents) for your reference.

> I think the docstring should explain this so that users are less likely to use the wrong one.

Sure, I've specified the behavior in the docstring now.